### PR TITLE
[9.x] Document Vite refresh option

### PR DESCRIPTION
**Pull request by timacdonald**
See: https://github.com/laravel/vite-plugin/pull/43, https://github.com/laravel/vite-plugin/pull/55

### CODE REVIEW

File: vite.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
   - [Inertia](#inertia)
   - [URL Processing](#url-processing)
 - [Working With Stylesheets](#working-with-stylesheets)
+- [Working With Views](#working-with-views)
```
Comment (jessarcher) [inline vite.md:17]: What do you think about "Working With Blade"?

(I'd have added a suggestion, but it requires the anchor to be updated as well)

Comment (timacdonald) [reply vite.md:17]: I put Views as I felt that "PHP" was too generic. The project might be 99% PHP. Views felt more specific and Blade felt too specific.

So I landed in the middle. Happy to change it if you still feel that "PHP" would be better?

Comment (jessarcher) [reply vite.md:17]: My only concern with "Views" is the term can apply to JavaScript views, which this configuration is not intended for.

Comment (timacdonald) [reply vite.md:17]: Maybe "Vues" just to really lean into it?

Yea, I get that. What do you think about leaning to the Blade direction then?

"Working with Blade"

Comment (jessarcher) [reply vite.md:17]: That's what I first thought, but it doesn't really cover the routes.

Is "Working with Blade and Routes" too much?

Comment (timacdonald) [reply vite.md:17]: I can get behind this TBH

**Pull request merged**